README.md

```diff
@@ -156,7 +156,7 @@ palomad tx staking create-validator \
       --moniker="$MONIKER" \
       --website="https://www.example.com" \
       --details="<enter a description>" \
-      --chain-id=paloma-testnet-8 \
+      --chain-id=paloma-testnet-9 \
```

Comment:
There is still a `CHAIN_ID=paloma-testnet-7` far below.

Reply:
updated. Side note that addresses in the setup script are also outdated